offgridplanner/users/adapters.py

```diff
@@ -17,6 +25,75 @@ class AccountAdapter(DefaultAccountAdapter):
     def is_open_for_signup(self, request: HttpRequest) -> bool:
         return getattr(settings, "ACCOUNT_ALLOW_REGISTRATION", True)
 
+    def send_mail(self, template_prefix: str, email: str, context: dict) -> None:
+        request = globals()["context"].request
+        ctx = {
+            "request": request,
+            "email": email,
+            "current_site": get_current_site(request),
+        }
+        ctx.update(context)
+        msg = self.render_mail(template_prefix, email, ctx)
+        if settings.USE_CUSTOM_SENDMAIL is True:
+            self.send_custom_mail(
+                to_email=msg.to,
+                subject=msg.subject,
+                message=msg.body,
+            )
+        else:
+            msg.send()
+
+    def send_custom_mail(self, to_email, subject, message):
```

Comment:
This is the send_mail code from OpenPlan, it uses smtplib directly as the django smtp default seem not to connect succesfully to office365